build-parent/pom.xml

```diff
@@ -147,27 +147,13 @@
     </properties>
 
     <dependencyManagement>
-        <!-- NB: Why are these here and not with the other dependencies? -->
         <dependencies>
+            <!-- force upgrade jandex from 3.1.6 to 3.2.2 for compatibility with smalrye 4.24-->
+            <!-- NB: this will likely be unnecessary after upgrading to Quarkus 3.14+ -->
             <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-core</artifactId>
```

Comment:
I: Moved to build parent so they could be chared with `aissemble-enforcer-extension` along with some of the apache-commons libraries